lessons/intro/distribution/index.md

```diff
@@ -105,25 +109,46 @@ Další soubory lze přidat pomocí souboru `MANIFEST.in`, dle [dokumentace].
 
 [dokumentace]: https://docs.python.org/3/distutils/sourcedist.html#specifying-the-files-to-distribute
 
-V tomto případě takto:
+V našem případě bude `MANIFEST.in` vypadat takto:
 
 ```
 include LICENSE
 ```
-```bash
-(env)$ python3 setup.py sdist
+
+Při dalším spuštění už `setup.py` přidá i soubor `LISENSE`.
+To můžete zkontrolovat i ve výsledném archivu.
+
+```console
+(env)$ python setup.py sdist
 ...
 hard linking LICENSE -> isholiday-0.1
 hard linking MANIFEST.in -> isholiday-0.1
 hard linking README -> isholiday-0.1
 ...
 ```
 
+Hotový balíček pak můžete nainstalovat pomocí `pip`.
```

Comment:
nástroje/příkazu `pip` (jinak by to bylo pomocí pipu a skloňovat to moc nechceme)